src/ui/component/viewers/videoViewer/view.jsx

```diff
@@ -0,0 +1,79 @@
+// @flow
+import React, { createRef, useEffect } from 'react';
+import { stopContextMenu } from 'util/context-menu';
+import videojs from 'video.js';
+import 'video.js/dist/video-js.css';
+import isUserTyping from 'util/detect-typing';
+
+const SPACE_BAR_KEYCODE = 32;
+const VIDEO_JS_OPTIONS = {
+  autoplay: true,
+  controls: true,
+  preload: 'auto',
+  playbackRates: [0.5, 1, 1.25, 1.5, 2],
```

Comment:
Add 0.25, 0.75, and 1.1 (assuming no or minimal UI cost or other costs)

YouTube does 0.25 and 0.75 and I've found 1.1 is not very perceptible but still saves meaningful time